onnxruntime/test/util/test_utils.cc

```diff
@@ -183,7 +183,7 @@ void SparseIndicesChecker(const ONNX_NAMESPACE::TensorProto& indices_proto, gsl:
         ASSERT_STATUS_OK(utils::UnpackInitializerData(indices_proto, model_path, unpack_buffer));
         ind_span = ReinterpretAsSpan<const int64_t>(gsl::make_span(unpack_buffer));
       } else {
-        ind_span = gsl::make_span(indices_proto.int64_data().cbegin(), indices_proto.int64_data().cend());
+        ind_span = gsl::make_span(&(*indices_proto.int64_data().cbegin()), &(*indices_proto.int64_data().cend()));
```

Comment:
I think if you get a ptr and then the size, it would be good as well. `gsl::make_span(int64_data(), int64_size())` or something to that effect. The reason it is possible is that int64 data is contiguous.

Reply:
+1. I believe int64_data() returns a RepeatedField which has data() and size() member functions. that would be preferable to dereferencing cend().